### PR TITLE
Release 9.1.3

### PR DESCRIPTION
**Pull request by r0mant**
Should only merge after https://github.com/gravitational/teleport/pull/12340.

### CODE REVIEW

**Review by jakule — APPROVED** [review]

**Review by russjones — APPROVED** [review]

**Review by marcoandredinis — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 9.1.3
+
+This release of Teleport contains multiple improvements and bug fixes.
+
+* Fixed issue with some MySQL clients not being able to connect to MySQL 8.0 servers. [#12340](https://github.com/gravitational/teleport/pull/12340)
+* Fixed multiple conditions that could lead to SSH sessions freezing. [#12286](https://github.com/gravitational/teleport/pull/12286)
+* Fixed issue with `tsh db ls` failing for leaf clusters. [#12320](https://github.com/gravitational/teleport/pull/12320)
+* Fixed scenario in which internal cache could potentially become unhealthy. [#12251](https://github.com/gravitational/teleport/pull/12251)
```
Comment (ptgott) [inline CHANGELOG.md:10]: I would replace "internal cache" with "Teleport's internal cache" or "`$SERVICE_NAME`'s internal cache" (as applicable).

Comment (ptgott) [inline CHANGELOG.md:10]:
```suggestion
* Fixed a scenario in which internal cache could potentially become unhealthy. [#12251](https://github.com/gravitational/teleport/pull/12251)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+* Improved performance when opening new Application Access sessions. [#12300](https://github.com/gravitational/teleport/pull/12300)
+* Added additional flags to `teleport configure` command. [#12267](https://github.com/gravitational/teleport/pull/12267)
```
Comment (ptgott) [inline CHANGELOG.md:12]:
```suggestion
* Added flags to the `teleport configure` command. [#12267](https://github.com/gravitational/teleport/pull/12267)
```

**Review by ptgott — APPROVED** [review]
Approved with some nits

**Pull request merged**